### PR TITLE
21 admin page dynamic

### PR DESCRIPTION
**Pull request by BredeFK**
# Changes
* Courses show in admin/courses
* Assignment is dsplayed in admin/assignments
* Both is displayed in coorect order on /courses and /assignments  and also /admin/index
* Added error handling for courses
* Script is fixed
* Changed `fmt.Println` to `log.Println`

### CODE REVIEW

**Comment by BredeFK** [comment]
[#21 - Admin Page Dynamic](https://trello.com/c/J8GQvTCt/21-21-admin-page-dynamic)

File: controller/admin.go [hunk 1]
```diff
@@ -186,7 +204,10 @@ func AdminAssignmentGET(w http.ResponseWriter, r *http.Request) {
 
 	assignmentRepo := model.AssignmentRepository{}
 
-	assignments, err := assignmentRepo.GetAll()
+	// Get all assignments to user in sorted order
+	assignments, err := assignmentRepo.GetAllToUserSorted(session.GetUserFromSession(r).ID)
+	fmt.Println(assignments)
```
Comment (JohanAanesen) [inline controller/admin.go]: remove

File: model/course.go [hunk 2]
```diff
@@ -25,17 +25,18 @@ type Course struct {
...
 
-	//Create an empty courses array
+	// Create an empty courses array
 	var courses Courses
 
-	rows, err := db.GetDB().Query("SELECT course.* FROM course INNER JOIN usercourse ON course.id = usercourse.courseid WHERE usercourse.userid = ?", userID)
+	// Query gets all courses to user with userid: userID and sort them by year and semester
+	rows, err := db.GetDB().Query("SELECT course.* FROM course INNER JOIN usercourse ON course.id = usercourse.courseid WHERE usercourse.userid = ? ORDER BY course.year DESC, course.semester DESC", userID)
 	if err != nil {
 		fmt.Println(err.Error()) // TODO : log error
```
Comment (JohanAanesen) [inline model/course.go]: log.println instead

**Comment by JohanAanesen** [comment]
[#21 - Admin Page Dynamic](https://trello.com/c/J8GQvTCt/21-21-admin-page-dynamic)

**Pull request merged**